tools/frr-reload.py

```diff
@@ -2156,21 +2161,33 @@ def compare_context_objects(newconf, running):
         lines_to_configure = []
 
         if lines_to_del:
-            print("\nLines To Delete")
-            print("===============")
+            if not args.test_reset:
+                print("\nLines To Delete")
+                print("===============")
 
             for (ctx_keys, line) in lines_to_del:
 
                 if line == "!":
                     continue
 
-                cmd = "\n".join(lines_to_config(ctx_keys, line, True))
+                nolines = lines_to_config(ctx_keys, line, True)
+
+                # For topotests leave these lines in (don't delete)
+                # [chopps: why is "log file" more special than other "log" commands?]
+                if args.test_reset:
+                    nolines = [x for x in nolines if "debug" not in x and "log file" not in x]
+                    if not nolines:
+                        continue
+
+                cmd = "\n".join(nolines)
                 lines_to_configure.append(cmd)
+                # assert "log file" not in cmd, "cmd: {} line: {}".format(cmd, line)
```

Comment:
is this ... leftover debugging code? or just a comment that's sort of opaque?

Reply:
leftover debug.. :)